students/BrandonHenson/Lesson9/Mailroom_5.py

```diff
@@ -0,0 +1,138 @@
+# Brandon Henson
+# 5/17/18
+# Lesson 09 mailroom_5.py
+# Refactoring
+
+
+class Donor(object):
```

Comment:
In Python 3, this can just be
```
class Donor:
```
(no need to explicitly inherit from `object` when defining a class)